content/en/docs/languages/ruby/quickstart.md

```diff
@@ -6,7 +6,7 @@ weight: 10
 
 ### Prerequisites
 
-- Ruby version 2 or higher
+- Ruby version 2.x. gRPC does not support Ruby version 3 at this time.
```

Comment:
grpc does actually support ruby 3.0. This comment about keyword args is I believe outdated

Reply:
Great point! I have actually been using grpc with Ruby 3 since this was opened.